frontend/src/metabase/entities/collections/collections.ts

```diff
@@ -3,7 +3,7 @@ import { createSelector } from "@reduxjs/toolkit";
 
 import { GET } from "metabase/lib/api";
 import { createEntity, undo } from "metabase/lib/entities";
-import * as Urls from "metabase/lib/urls";
+import * as Urls from "metabase/lib/urls/collections";
```

Comment:
Not sure I understand why is that change here. :)

Reply:
I introduced some circular deps. But I will do that in a second PR